### ISSUE TITLE
grafana docs could be improved

### ISSUE THREAD

**Issue by huntergregory** [microsoft/retina#383]
Tried following documentation for setting up grafana and the steps felt incomplete and a bit confusing.
 
This section on [Unmanaged Prometheus/Grafana](https://retina.sh/docs/installation/prometheus-unmanaged)
![image](https://github.com/microsoft/retina/assets/42728408/13150bf1-21c5-4f98-ba3f-385447450e8b)

sends you to [Configuring Grafana](https://retina.sh/docs/installation/grafana):
![image](https://github.com/microsoft/retina/assets/42728408/a5998493-fc2c-4fe8-b31f-c529779cd251)

### Fixes
1. I would move all Grafana details to the latter page. The steps are hard to follow / incomplete right now.
2. Should include screenshots for logging in.
3. Also specify command for getting the admin username.
4. Also specify command for port-forwarding grafana.
a. `kubectl port-forward -n kube-system svc/prometheus-grafana 9999:80`
b. go to localhost:9999 in browser
5. The screenshot on latter page should be replaced imo since it is not the grafana from the helm installation).
6. formatting is a bit off on latter page

**Comment by TropicolX** [comment]
Hi @huntergregory! Can I be assigned to this issue?

**Comment by huntergregory** [comment]
Sure! Thank you!
